dockerized

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euxo pipefail
+
+# We always want the latest image
+docker pull ubuntu:20.04
+
+# CACHEBUST means this will apply the updates once a day
+docker build . -t distributions --build-arg CACHEBUST=`date --iso-8601=date` --build-arg USER_UID=$(id -u "$USER")
+
+# We use host networking as the build process assumes a fairly long-lived ipfs
+# node has the CIDs (we give them to the collab cluster to pin) 
+docker run -it --network host -v `pwd`:/build distributions "$@"
```

Comment:
```suggestion
docker run --rm -it --network host -v `pwd`:/build distributions "$@"
```
There should not be any need to keep the exited container around after the build, so clean it up after done with docker container.